src/main/java/com/commercetools/sync/producttypes/utils/PlainEnumUpdateActionsUtils.java

```diff
@@ -0,0 +1,67 @@
+package com.commercetools.sync.producttypes.utils;
+
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.models.EnumValue;
+import io.sphere.sdk.producttypes.ProductType;
+import io.sphere.sdk.producttypes.commands.updateactions.ChangePlainEnumValueLabel;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+import static com.commercetools.sync.commons.utils.CommonTypeUpdateActionUtils.buildUpdateAction;
+import static java.util.stream.Collectors.toList;
+
+public final class PlainEnumUpdateActionsUtils {
+    /**
+     * Compares all the fields of an old {@link EnumValue} and a new {@link EnumValue} and returns a list of
+     * {@link UpdateAction}&lt;{@link ProductType}&gt; as a result. If both {@link EnumValue} have identical fields,
+     * then no update action is needed and hence an empty {@link List} is returned.
+     *
+     * @param attributeDefinitionName the attribute definition name whose plain enum values belong to.
+     * @param oldEnumValue            the enum value which should be updated.
+     * @param newEnumValue            the enum value where we get the new fields.
+     * @return A list with the update actions or an empty list if the enum values are
+     *         identical.
+     */
+    @Nonnull
+    public static List<UpdateAction<ProductType>> buildActions(
+        @Nonnull final String attributeDefinitionName,
+        @Nonnull final EnumValue oldEnumValue,
+        @Nonnull final EnumValue newEnumValue) {
+
+        return Stream
```

Comment:
(just a question) don't you think it looks better this way:

````java
return Stream.of(buildChangeLabelAction(attributeDefinitionName, oldEnumValue, newEnumValue))
             .filter(Optional::isPresent)
             .map(Optional::get)
             .collect(toList());
````

Reply:
c3d86f78c7672ae9811d17d51c2cc8fc3d2d5a13